### PR TITLE
Opentelecom smtp fax error notifications

### PR DESCRIPTION
**Pull request by jamesaimonetti**
includes and resolves #3323

### CODE REVIEW

File: applications/fax/src/fax_smtp.erl [hunk 1]
```diff
@@ -280,6 +280,7 @@ maybe_system_report(#state{faxbox='undefined', account_id='undefined'}=State) ->
         'false' -> 'ok'
     end;
 maybe_system_report(State) ->
+    send_outbound_smtp_fax_error(State),
```
Comment (lazedo) [inline applications/fax/src/fax_smtp.erl]: @jamesaimonetti fax attachment error will be sent here. this is not desirable. you can put behind a configuration flag (false by default) and would prefer to put that in the `maybe_faxbox_log` clauses

Comment (jamesaimonetti) [inline applications/fax/src/fax_smtp.erl]: @lazedo noted, will update

**Comment by jamesaimonetti** [comment]
@lazedo updated, what do you think? @kalda341 thoughts? care to test it on your setup?

**Comment by lazedo** [comment]
@jamesaimonetti i think the solution you found with the `has_smtp_errors` is great and would make the `report_smtp_errors` defaulting to `true` possible

**Comment by kalda341** [comment]
Looks good to me! Finding time to test is quite tricky for me at the moment, so the PR might have to wait a while if you need it tested. I am happy with it if it works for you though :)

**Comment by jamesaimonetti** [comment]
@lazedo ok

@kalda341 sounds good, I think we'll go for it.

**Pull request merged**